### PR TITLE
implement blast from the past fixes

### PR DESCRIPTION
**Pull request by jamesbradleym**
**_I affirm:_**
- [x] I understand that if I do not agree to the following points by completing the checkboxes my PR will be ignored.
- [x] I understand I should leave resolving conversations to the LandSandBoat team so that reviewers won't miss what was said.
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/LandSandBoat/server/blob/base/CODE_OF_CONDUCT.md).
- [x] I have _**tested my code and the things my code has changed**_ since the last commit in the PR and will test after any later commits.

## What does this pull request do?

Fixes some inconsistencies with retail for quest Blast From The Past

[Capture](https://www.dropbox.com/scl/fo/fhgjn0edw5ivqki2n0b2a/h?rlkey=pgttbon5mp76p76k7dq6stxwz&dl=0)

## Steps to test these changes

Run through quest Blast From The Past, speaking with Koru-Moru, Yoran-Oran, Tokaka, Maan-Pokuun, and Bonchacha

### CODE REVIEW

File: scripts/quests/windurst/Blast_from_the_Past.lua [hunk 1]
```diff
@@ -144,12 +144,13 @@ quest.sections =
     {
         check = function(player, status, vars)
             return status == xi.questStatus.QUEST_COMPLETED and
-                not xi.quest.getMustZone(player, xi.questLog.WINDURST, xi.quest.id.windurst.NOTHING_MATTERS)
+                xi.quest.getMustZone(player, xi.quest.log_id.WINDURST, xi.quest.id.windurst.NOTHING_MATTERS)
```
Comment (claywar) [inline scripts/quests/windurst/Blast_from_the_Past.lua]: This needs to be rebased with current code, the enum change here is deprecated

**Review by claywar — CHANGES_REQUESTED** [review]

File: scripts/quests/windurst/Blast_from_the_Past.lua (hunk 1 above)
Comment (jamesbradleym) [reply scripts/quests/windurst/Blast_from_the_Past.lua]: Thanks! - Updated to `xi.questLog`

Presumably the examples in lua_baseentity.cpp should be updated? Or am I reading that wrong?

**Comment by jamesbradleym** [comment]
@claywar Anything else needed here, Clay? :)

**Comment by jamesbradleym** [comment]
Closing this PR to reduce open PRs and prevent staleness—will revisit.

**Pull request closed without merging**